lib/elixir/lib/string.ex

```diff
@@ -286,7 +286,7 @@ defmodule String do
   @typedoc "Pattern used in functions like `replace/4` and `split/3`"
   @type pattern :: t | [t] | :binary.cp()
 
-  @conditional_mappings [:greek]
+  @conditional_mappings [:greek, :turkish]
```

Comment:
I believe we should it `:turkic` as it applies to turkic languages (az, tr) but it has been a while since I have been through Unicode documents.